common/script/cron.js

```diff
@@ -41,7 +41,7 @@ function sanitizeOptions (o) {
     timezoneOffset = timezoneOffsetDefault;
   }
 
-  let now = o.now ? moment(o.now).zone(timezoneOffset) : moment().zone(timezoneOffset);
+  let now = o.now ? moment(o.now).utcOffset(timezoneOffset) : moment().utcOffset(timezoneOffset);
```

Comment:
This looks like it's meant to address the fact that Moment has deprecated `zone` and keeps blaring that fact into our server logs. Is now not a good time to do that?